src/components/styles/Theme.tsx

```diff
@@ -285,16 +285,15 @@ export const zopaTheme: AppTheme = {
 const ThemeContext = React.createContext<AppTheme>(zopaTheme);
 ThemeContext.displayName = 'ThemeContext';
 
-export function useThemeContext() {
-  const context = React.useContext(ThemeContext);
-  if (!context) {
-    throw new Error('No theme provided');
-  }
-
-  return context;
-}
+export const useThemeContext = () => React.useContext(ThemeContext);
 
-export const ThemeProvider = ({ children, theme }: { children: React.ReactNode; theme: Partial<AppTheme> }) => {
-  const mergedTheme: AppTheme = { ...zopaTheme, ...theme };
+export const ThemeProvider = ({
+  children,
+  theme,
+}: {
+  children: React.ReactNode;
+  theme: Partial<AppTheme> | undefined;
```

Comment:
```suggestion
  theme?: Partial<AppTheme>;
```

what do you think?

Reply:
Yeah I can see that kind of rationale for this api. Lets roll with it and then we can see how it works out in practice. Given we are the main consumers (its a closed api) - we can maybe revisit